tests/unit/lms/services/lti_grading/_v13_test.py

```diff
@@ -8,16 +8,28 @@
 
 
 class TestLTI13GradingService:
-    SERVICE_URL = "http://example.com/service_url"
-
-    def test_read_lti_result(self, svc, response, ltia_http_service):
+    @freeze_time("2022-04-04")
+    @pytest.mark.parametrize(
+        "service_url,results_url",
+        [
+            ("https://lms.com/lineitems", "https://lms.com/lineitems/results"),
+            (
+                "https://lms.com/lineitems?param=1",
+                "https://lms.com/lineitems/results?param=1",
+            ),
+        ],
+    )
```

Comment:
I think you could just use the more difficult example here, as it's a superset of everything going on in the other example.